appveyor.yml

```diff
@@ -0,0 +1,62 @@
+# This file was automatically generated by conda-smithy. To update a component of this
+# file, make changes to conda-forge.yml and/or recipe/meta.yaml, and run
+# "conda smithy rerender".
+
+environment:
+
+  CONDA_INSTALL_LOCN: "C:\\conda"
+
+  # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
+  # /E:ON and /V:ON options are not enabled in the batch script intepreter
+  # See: http://stackoverflow.com/a/13751649/163740
+  CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
+
+  # We set a default Python version for the miniconda that is to be installed. This can be
+  # overridden in the matrix definition where appropriate.
+  CONDA_PY: "27"
+  BINSTAR_TOKEN:
+    # The BINSTAR_TOKEN secure variable. This is defined canonically in conda-forge.yml.
+    secure: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+
+  matrix:
+    - TARGET_ARCH: x86
+
+    - TARGET_ARCH: x64
```

Comment:
This is pretty strange. We didn't have an AppVeyor file before and am not sure that this one is even right. Thoughts?

cc @pelson